pkg/probe/http/http.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package http
+
+import (
+	"net"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/probe"
+
+	"github.com/golang/glog"
+)
+
+var client = &http.Client{}
```

Comment:
we've swapped one set of globals for another, but it's fine.  At least they are concentrated now and would be easy to change if it becomes a problem.